.github/CODEOWNERS

```diff
@@ -38,8 +38,6 @@
 /tests/ecosystem/upgrade/ @red-hat-storage/purple-squad
 /tests/e2e/flowtest/ @red-hat-storage/magenta-squad
 /tests/e2e/lifecycle/ @red-hat-storage/magenta-squad
-/tests/e2e/logging/ @red-hat-storage/magenta-squad
-/tests/e2e/registry/ @red-hat-storage/magenta-squad
 /tests/e2e/workloads/ @red-hat-storage/magenta-squad
 /tests/e2e/performance/ @red-hat-storage/grey-squad
```

Comment:
shouldn't we add **kcs** folder here?

Reply:
Yes, we should add kcs as well. 
We will have to add system test as well once we have the system tests PRs merged. I will add along with it.